providers/freebsd/sysctl_freebsd_test.go

```diff
@@ -0,0 +1,94 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+//go:build freebsd
+
+package freebsd
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestArchitecture(t *testing.T) {
+	arch, err := architecture()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	assert.NotEmpty(t, arch)
+}
+
+func TestBootTime(t *testing.T) {
+	bootTime, err := bootTime()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	// Apply a sanity check. This assumes the host has rebooted in the last year.
```

Comment:
Can we shell out to `uptime` and get a value to use?

Reply:
I created a [PR on the forked repo](https://github.com/redanthrax/go-sysinfo/pull/1) that addresses your request (awaiting approval). It's a bit overkill for a test, but there's no other structured way to get the uptime apart from `sysctl` or adding more CGO.

Reply:
That looks reasonable after clean-up; not overkill at all.